backend/src/envPlatform/schema.bal

```diff
@@ -0,0 +1,94 @@
+
+type TreeRemovalApplication record { 
+     int numberOfVersions;
+     int versionNumber;
+     string applicationId;
+     string title;
+     Date applicationCreatedDate;
+     Date removalDate;
+     string reason;
+     string type;
```

Comment:
We can't have a variable named **type** it's a reserved word.

Reply:
I will change it to applicationType. Since it is autogenerated We have to change the naming conventions.

Reply:
+1, @yashodgayashan mentioned he will do the changes